CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ feel free to ask us and community.
 
 ### Bug Fixes
 
+* fixed missing call `transformer.from()` in case column is NULL ([#3395](https://github.com/typeorm/typeorm/issues/3395))
```

Comment:
this is a bit breaking since users will have `null` now instead of value and their transform methods can give them `cannot read property x of null` now. I'm okay to merge it to master, I just aware you about it.